WalletWasabi.Fluent/ViewModels/WalletManagerViewModel.cs

```diff
@@ -94,7 +94,7 @@ private WalletViewModelBase OpenWallet(WalletManager walletManager, UiConfig uiC
 
 			if (!walletManager.AnyWallet(x => x.State >= WalletState.Started && x != walletViewModel.Wallet))
 			{
-				walletViewModel.OpenWalletTabs();
+				WalletViewModel.OpenWalletTabs();
```

Comment:
What does this change do?

Reply:
After marking `public void OpenWalletTabs()`  as static, it cannot be accessed with an instance reference.

Reply:
This is a bit weird because `public static void OpenWalletTabs()` is not implemented at the moment. I expect that `OpenWalletTabs` will not be `static` ultimately. Probably OK for now.

Reply:
That makes sense, I reverted that.